ci-operator/step-registry/mco/test/OWNERS

```diff
@@ -0,0 +1,16 @@
+approvers:
+  - Xia-Zhao-rh
```

Comment:
These are all QE folks, shouldn't we own our own steps?

(and again back to the original question of whether we should be using QE workflows)